packages/core/src/components/progress/progress.stories.ts

```diff
@@ -0,0 +1,71 @@
+import { htmlMatrix, Meta, omit, Story } from '../../../../../docs';
+import { Progress, ProgressProps } from './progress.story';
+
+const size = ['regular', 'large'] as const;
+
+export default {
+  title: 'Core/Progress',
+  component: Progress,
+  argTypes: {
```

Comment:
This now has unneeded `aria-valuetext` in table, better to:

```suggestion
  argTypes: {
    ...omit<ProgressProps>('aria-valuetext'),
```

Reply:
Nice catch, didn't notice that, changed, thanks!